### ISSUE TITLE
Let user know who contributed to the document and what each author added

### ISSUE THREAD

**Issue by ImaCrea** [nextcloud/notes#758]
**Is your feature request related to a problem? Please describe.**
At the moment, when a user opens a document she can't know who wrote what's there. 

**Describe the solution you'd like**
A section should show easily how many people have contributed to the document so far. If they have accounts, it should show who they are. And a checkbox should allow to colorize the content in order to easily let the user see who contributed what.

**Describe alternatives you've considered**
At the moment people write their name directly in the document to let others know what they added. It's ok but it quickly becomes difficult to manage and disturb the overall experience.

**Additional context**
I'm taking the time to write here this issue as it's a feature many users we serve love on etherpad and which they asked us to have in Nextcloud Notes. (Actually, because of this feature lacking some asks us to integrate etherpad in NC, which is not so ideal.)

Thanks for your great work folks, love NC! 🤗

**Comment by ImaCrea** [comment]
There's been some improvement here by showing different contribution colored while people are editing together the doc. It'd be even better if this is information was resilient over time as described in the feature request I wrote.. last year. :)

**Comment by joshtrichards** [comment]
Are you sure Notes is the best solution here? Almost sounds like a good use case for the Collectives app.

**Comment by N07070** [comment]
I would agree that this would be very useful. It could make Etherpad obsolete. It might also be useful for Collectives, but sometimes you just need a quick note taking, a full Collective  is overkill for that.

**Comment by juliusknorr** [comment]
Since both notes and collectives use the text editor, this is probably the issue to upvote: https://github.com/nextcloud/text/issues/3731

Once re-implemented in text, it will become available in notes as well.

Will close this here as tracked upstream.
